### ISSUE TITLE
NTR: Nicotinamide adenine dinucleotide (NAD) low in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8403]
New term request Nicotinamide adenine dinucleotide (NAD) low in blood (elthat_171215053134):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000405
- label: Decreased circulating nicotinamide adenine dinucleotide concentration
- parents: [Abnormal circulating purine concentration (HP:0004352)](https://hpo.jax.org/app/browse/term/HP:0004352)
- pmid: 
- def: The concentration of nicotinamide adenine dinucleotide in the blood circulation is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8403
- synonyms:
